src/js/components/shepherd-element.svelte

```diff
@@ -79,7 +79,7 @@
         }
         // Backward tab
         if (e.shiftKey) {
-          if (document.activeElement === firstFocusableElement) {
+          if (document.activeElement === firstFocusableElement || document.activeElement.hasAttribute("data-popper-placement")) {
```

Comment:
Would it perhaps make more sense to check for a class of `shepherd-element`?

Reply:
That would be perfect! Thanks for the suggestion